answerking_app/tests/test_integration/snapshots/snap_test_products.py

```diff
@@ -439,6 +432,13 @@
     "type": "http://testserver/problems/error/",
 }
 
+snapshots["PutTests::test_put_retired_product_returns_forbidden 1"] = {
+    "detail": "This object has already been retired, unretire it before updating it",
+    "status": 403,
```

Comment:
Is 403 the right status code to use? is that not for a forbidden URL or something?

Reply:
Yeah, makes sense to me but whichever number was agreed, no idea tbh.
@JoeCSykes 410?

Reply:
Yeah I guess it does make sense actually as you shouldnt be able to reach that endpoint if its retired.

Reply:
but it looks like 410 gone is still in the contract for calls to an endpoint with retired product id

Reply:
so we are using "410" for objects that have been retired